src/cmap/conn.rs

```diff
@@ -22,8 +22,12 @@ use crate::{
     compression::Compressor,
     error::{load_balanced_mode_mismatch, Error, ErrorKind, Result},
     event::cmap::{
-        CmapEventEmitter, ConnectionCheckedInEvent, ConnectionCheckedOutEvent,
```

Comment:
Looks like somehow the rustfmt I run on nvim buffer save is not the correct version of rustfmt.